packages/block-library/src/navigation/style.scss

```diff
@@ -142,6 +142,23 @@
 		}
 	}
 
+	&.items-justification-left {
```

Comment:
The nesting for the `> ul` rules is unnecessary since you can just do &.items-justification-left > ul`. Same for the next couple of rules.

Reply:
Yeh not sure why I did that. I'm breaking my own rules doing that! 

Thanks. All fixed.